core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -2960,7 +2960,7 @@ private RecordCursorFactory generateSelectChoose(QueryModel model, SqlExecutionC
                 QueryColumn qc = columns.getQuick(i);
                 if (
                         !Chars.equals(metadata.getColumnName(i), qc.getAst().token) ||
-                                qc.getAlias() != null && !(Chars.equals(qc.getAlias(), qc.getAst().token))
+                                qc.getAlias() != null && !(Chars.equalsIgnoreCase(qc.getAlias(), qc.getAst().token))
```

Comment:
nit: unrelated with this PR, but this way the expression will be more obvious
```suggestion
                                (qc.getAlias() != null && !Chars.equalsIgnoreCase(qc.getAlias(), qc.getAst().token))
```